main.c

```diff
@@ -69,6 +69,22 @@ int main(void)
     MSS_TIM64_enable_irq();
     NVIC_SetPriority(Timer1_IRQn, 1);
 
+    /* load citi conf on startup */
+    uint32_t cfg_id_32 = 0;
+    mem_read(NVM_CITIROC_CONF_NO, &cfg_id_32);
+    uint8_t cfg_id = (uint8_t)(cfg_id_32);
+    uint8_t *nvm_cfg_addr =
+    (uint8_t *)(NVM_ADDR+CITIROC_OFFSET+(cfg_id*CITIROC_LEN));
```

Comment:
This line can be indented by a TAB or two. Alternatively, you could leave the `nvm_cfg_addr` pointer declaration without an initialization and assign to the pointer on the next line (this might look cleaner).